micro-rdk/src/common/app_client.rs

```diff
@@ -187,12 +189,14 @@ impl AppClient {
     pub(crate) fn initiate_signaling(
         &self,
         rpc_host: String,
+        heartbeats_allowed: String,
     ) -> impl Future<Output = Result<AppSignaling, AppClientError>> {
         let (sender, receiver) = async_channel::bounded::<Bytes>(1);
         let r = self.grpc_client.build_request(
```

Comment:
the fucntion returns the request so it make more sense to mutate the request by adding the header after